lib/cli.js

```diff
@@ -97,10 +98,23 @@ const getArguments = (schema = webpackSchema) => {
 	const getDescription = path => {
 		for (const { schema } of path) {
 			if (schema.cli && schema.cli.helper) continue;
+			if (schema.cliDescription) return schema.cliDescription;
```

Comment:
since this is cli specific this should be moved into `schema.cli.description`

Reply:
Makes sense 👍🏻